walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -17,6 +17,20 @@
 import java.util.Collection;
 
 public final class FindMeetingQuery {
+  /**
+    Returns whether or not an event is attended by the required attendees from the Meeting Request
```

Comment:
[for javadoc comments](https://google.github.io/styleguide/javaguide.html#s7-javadoc):

- each comment line between the `/**`-line and the `*/`-line starts with a `*` aligned with the first `*` of the `/**`-line
- the summary fragment (that explains what the function does) is separated by a blank line (i.e., one that only has a `*`) from the `@param`/`@return` section below it.

Reply:
Added * in commit bfed837, added blank line with commit e1e1cab